ups/tmv.table

```diff
@@ -0,0 +1,7 @@
+setupRequired(scons)
+envPrepend(PATH,${PRODUCT_DIR}/bin)
+envPrepend(LD_LIBRARY_PATH,${PRODUCT_DIR}/lib)
+envPrepend(LIBRARY_PATH,${PRODUCT_DIR}/lib)
```

Comment:
LIBRARY_PATH, C_INCLUDE_PATH, and CPATH are not typically used.  Are you sure they are necessary?